controllers/community_controller.js

```diff
@@ -1,25 +1,102 @@
 // Dependencies
 
 // Models
+const Community = require('../models/CommunityModel');
+const Profile = require('../models/ProfileModel');
+
+/*
+1) Create -- first pass done
+2) Delete -- first pass done
+3) Get list -- first pass done
+4) Get specific -- first pass done
+5) update -- first pass done
+*/
+
+/*
+{
+  name: String,
+  description: String,
+  headerPicture: String, // TODO: change to actual image storage
+  admins: [String],
+}
+*/
+
+async function getProfileIdsFromUsernames(usernames) {
```

Comment:
Since this same function exists in a couple places, we could move this to a file full of utility functions if we have others that are repeated in multiple places, that way we can just import them into files where we need to use them. Not urgent right now, but as we get other functions that we use in multiple places, we'll want to place them in a central file and import them